pipeline.py

```diff
@@ -70,20 +70,36 @@
     description="InstructLab pipeline",
 )
 def ilab_pipeline(
+    # Model I/O
+    input_model_uri: str,
+    output_oci_model_uri: str,
+    output_oci_registry_secret: str,
+    output_model_name: str,
+    output_model_version: str,
```

Comment:
Optional: I think UX had made the comment that version name is clearer than just version so something like `output_model_version_name` may be clearer.